cli/internal/run/run_state.go

```diff
@@ -145,6 +145,11 @@ func (r *RunState) Close(terminal cli.Ui, filename string) error {
 	if r.Cached == r.Attempted && r.Attempted > 0 {
 		maybeFullTurbo = ui.Rainbow(">>> FULL TURBO")
 	}
+
+	if r.Attempted == 0 {
+		terminal.Output("") // Clear the line
```

Comment:
Using this pattern so that the ANSI formatting is limited to a single line of output.